### DISCUSSION TITLE
Bug in Filtering not working with Like operator

### DISCUSSION THREAD

**Discussion by xnohat** [Budibase/budibase#5016] [Help]
Hi,

I working with with newest Budibase 1.0.90. I found an bug in Filtering
Internal Budibase Database, filtering using Like operator not working.
![Screenshot 2022-03-23 014123](https://user-images.githubusercontent.com/4613808/159553379-0b1ca087-d953-4539-b8c7-6a1d820ce8d2.png)
But using Starts With operator is work
![Screenshot 2022-03-23 014233](https://user-images.githubusercontent.com/4613808/159553557-9b90a306-98b5-4fb2-8d46-de5b90920f34.png)

This also impact to Search Columns feature of Table Block , search only match with Starts With , not Like ( keyword is a part of string )

**Comment by melohagan** [comment]
Hi @xnohat 

I see you found the open GitHub issue https://github.com/Budibase/budibase/issues/3386

As mentioned, there are plans to remove the Like option for filters on the internal tables due to restrictions on CouchDB.

**Reply by xnohat** [reply]
Thanks, that a crucial technical problem.
